exchanges/bithumb/bithumb_test.go

```diff
@@ -539,7 +540,7 @@ func TestUpdateOrderExecutionLimits(t *testing.T) {
 	require.NoError(t, err, "GetOrderExecutionLimits must not error")
 
 	err = limit.Conforms(46241000, 0.00001, order.Limit)
-	assert.ErrorIs(t, err, order.ErrAmountBelowMin, "expected error: %v, but got: %v", order.ErrAmountBelowMin, err)
+	assert.ErrorIs(t, err, limits.ErrAmountBelowMin, "expected error: %v, but got: %v", limits.ErrAmountBelowMin, err)
```

Comment:
```suggestion
	assert.ErrorIs(t, err, limits.ErrAmountBelowMin)
```
Redundant error decoration since we'll get that output anyway if the test were to fail:

![image](https://github.com/user-attachments/assets/a8b805a6-bf80-4f94-b9f9-c089fe96f585)